src/extension.ts

```diff
@@ -114,9 +114,8 @@ export function activate(context: ExtensionContext) {
 
   //generic commands to open correct view on status bar click
   registerOpenViewsCommands(context, outputChannel);
-  commands.registerCommand(
-    "gitguardian.refreshQuota",
-    ggshieldQuotaViewProvider.refresh
+  commands.registerCommand("gitguardian.refreshQuota", () =>
+    ggshieldQuotaViewProvider.refresh()
   );
```

Comment:
I believe this does not change anything, I slightly preferred the previous version.

Reply:
This was actually (not sure why) the reason the button wasn't working anymore :(